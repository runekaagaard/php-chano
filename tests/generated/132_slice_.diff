001- abcdef
001+ Fatal error: Call to undefined method DtlIter::slice_() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/132_slice_.php on line 4