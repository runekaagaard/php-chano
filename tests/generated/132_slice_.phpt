--TEST--
A generated testfile for the "slice_" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'abcdefg')));
foreach ($items as $i) echo $i->input->slice_('-1');
--EXPECT--
abcdef