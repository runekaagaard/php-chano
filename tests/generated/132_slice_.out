Fatal error: Call to undefined method DtlIter::slice_() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/132_slice_.php on line 4